main.py

```diff
@@ -1,7 +1,31 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+
+    genres_to_create = ["Western", "Action", "Dramma"]
+
+    for genre in genres_to_create:
+        Genre.objects.create(name=genre)
+
+    actors_to_create = [("George", "Klooney"), ("Kianu", "Reaves"),
+                        ("Scarlett", "Keegan"), ("Will", "Smith"),
+                        ("Jaden", "Smith"), ("Scarlett", "Johansson")]
+
```

Comment:
```suggestion
    actors_to_create = [
                       ("George", "Klooney"), ("Kianu", "Reaves"),
                        ("Scarlett", "Keegan"), ("Will", "Smith"),
                        ("Jaden", "Smith"), ("Scarlett", "Johansson")
         ]

```
add breaking line after ```[```